StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement.swift

```diff
@@ -202,6 +212,7 @@ public final class EmbeddedPaymentElement {
             stpAssertionFailure(errorMessage)
             return .failed(error: PaymentSheetError.integrationError(nonPIIDebugDescription: errorMessage))
         }
+        analyticsHelper.log(event: .mcConfirmEmbedded)
```

Comment:
Same here, why not log first thing in this method?

Reply:
No strong reason on this one, I'll move it up.